modules/ROOT/pages/database-administration/aliases/remote-database-alias-configuration.adoc

```diff
@@ -131,92 +134,232 @@ dbms.security.key.name=key-name
-However, they can be accessed and terminated on the remote database when connecting with the same user.
+Note that only client-side SSL is supported.
+By default, remote aliases require a secured URI scheme such as `neo4j+s`.
+However, if you want to disable the secure URL scheme, you can set the driver setting `ssl_enforced` to `false`.
+====
+
+For example, you can use the following command to create a remote database alias with OIDC credential forwarding:
+
+[source, Cypher]
+----
+CREATE ALIAS `db1-remote-alias` FOR DATABASE `db1` AT "neo4j+s://location:7687" OIDC CREDENTIAL FORWARDING
+----
+
+[NOTE]
+====
+If a transaction modifies an alias (e.g. changing the database targeted on *DBMS B*), other transactions concurrently executing against that alias may be aborted and rolled back for safety.
+This prevents issues such as a transaction executing against multiple target databases for the same alias.
 ====
+
+== Connect to remote database aliases
+
+You can connect to a remote database alias the same way as you would connect to a standard database using any of the following options:
+
+* Connecting directly to the remote database alias.
+* Querying a remote database alias that you are not directly connected to using the Cypher link:{neo4j-docs-base-uri}/cypher-manual/current/clauses/use[`USE` clause]:
++
+[source, Cypher]
+----
+USE `db1-remote-alias` MATCH (n) RETURN *
+----
+
+* Connecting to a remote database alias as a home database.
+This needs to be set by an administrator.
+See xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-user-management[User Management] for more information.
++
+[source, Cypher]
+----
+ALTER USER alice SET HOME DATABASE `db1-remote-alias`
+----
+
+== Important notes
+
+When using remote database aliases, keep in mind that:
+
+* Remote alias transactions will not be visible in `SHOW TRANSACTIONS` on *DBMS A*.
+However, they can be accessed and terminated on the remote database when connecting with the same user.
+
+* Actions on the remote *DBMS B* are all attributed to the user configured for the remote database alias.
+In the case of using `STORED NATIVE CREDENTIALS`, the same credentials are used to connect to the remote DBMS regardless of which end-user made the query targeting the remote alias.
+This will result in the stored native user being logged in the audit trails on the remote DBMS for all queries using the remote database alias.
+When using `OIDC CREDENTIAL FORWARDING`, the actual end-user's credentials and permissions are used, resulting in per-user audit trails being logged on the remote DBMS.
+
+* When using a remote database alias with OIDC credential forwarding, the user needs to be logged in to the local DBMS with OIDC, otherwise there is no token to forward, and the access to the remote database will be denied with GQLSTATUS link:https://neo4j.com/docs/status-codes/current/errors/gql-errors/42NFF/[`42NFF`].
```

Comment:
Maybe, I didn't fully understand the concept of OIDC credential forwarding, but here a local DBMS is mentioned, and there is no local DBMS on the figure 2.

Reply:
I would say that it will clarify the situation, but Reneta mentioned earlier that the arrow 'remote connection' implies that the DBMS A is local and DBMS B is remote.